libp2p/network/swarm.py

```diff
@@ -231,15 +232,16 @@ def set_stream_handler(self, stream_handler: StreamHandlerFn) -> None:
         #   Reference: https://github.com/libp2p/go-libp2p-swarm/blob/8be680aef8dea0a4497283f2f98470c2aeae6b65/swarm.go#L124-L134  # noqa: E501
 
         # Close listeners
-        await asyncio.gather(
-            *[listener.close() for listener in self.listeners.values()]
-        )
-
-        # Close connections
-        await asyncio.gather(
-            *[connection.close() for connection in self.connections.values()]
-        )
-
+        # await asyncio.gather(
+        #     *[listener.close() for listener in self.listeners.values()]
+        # )
+
+        # # Close connections
+        # await asyncio.gather(
+        #     *[connection.close() for connection in self.connections.values()]
+        # )
+        self.manager.stop()
```

Comment:
I believe that this should be awaited:

https://async-service.readthedocs.io/en/latest/async_service.html#managers